client/views/profile.ejs

```diff
@@ -1,115 +1,118 @@
-    <!-- header section-->
-    <div class="bg" id="profile-bg">
-      <div class="profile-title">
-        <h1 id="cray-font">Welcome <%= user.fName %></h1>
-      </div>
+<body>
+  <%- include('./templates/nav')-%> <%- include('./templates/editAboutMeModal')-%>
+  <!-- header section-->
+  <div class="bg" id="profile-bg">
+    <div class="profile-title">
+      <h1 class="crayon-font">Welcome <%= user.fName %></h1>
     </div>
-
-    <!-- profile content section-->
-    <header class="profile-header quick-font">
-      <div class="container justify-content-center">
-        <% if (!user.emailVerified) { %>
-        <p class="mt-5 text-danger notify">
-          <i class="fa fa-exclamation-triangle" aria-hidden="true"> </i>Your email is unverified.
-          Some features are disabled until email is verified.
-        </p>
-        <% } %> <% if (user.userRole == 'partner' && locals.agency && (!agency.isVerified)) { %>
-        <p class="mt-5 text-danger notify">
-          <i class="fa fa-exclamation-triangle" aria-hidden="true"></i>
-          Wish card creation feature is disabled for your account. It will be enabled after agency
-          verification. For a faster verification,
-          <a href="mailto:donate.gifts.stacy@gmail.com" class="text-dark">click to email us</a>
-        </p>
-        <% } %> <% if (user.userRole == 'partner' && locals.agency && agency.isVerified) { %>
-        <div class="user-info my-5">
-          <h4 class="create-title pb-5">Create Wish Cards</h4>
-          <div class="row justify-content-center py-3 my-3">
-            <div class="col-md-4 col-xs-12 mb-4 text-md-center">
-              <p class="mb-4"><i class="fa fa-pencil"></i>Child has a specific wish item</p>
-              <a href="#wishCardForm" class="button button--inverted-green"> Create Wish Card </a>
-            </div>
-            <div class="col-md-4 col-xs-12 mb-4 text-md-center">
-              <p class="mb-4">
-                <i class="fa fa-bolt" aria-hidden="true"></i>Not sure which wish item to ask for?
-              </p>
-              <a href="/users/choose" class="link--green bdr-2">Our Suggested List </a>
-            </div>
-            <div class="col-md-4 col-xs-12 mb-4 text-md-center">
-              <p class="mb-4">
-                <i class="fa fa-question-circle" aria-hidden="true"></i>I have a question about this
-              </p>
-              <a href="mailto:donate.gifts.stacy@gmail.com" class="button button--inverted-green"
-                >Email Us</a
-              >
-            </div>
+  </div>
+
+  <!-- profile content section-->
+  <header class="profile-header quick-font">
+    <div class="container justify-content-center">
+      <% if (!user.emailVerified) { %>
+      <p class="mt-3 quick-font notify">
+        <i class="fa fa-exclamation-triangle" aria-hidden="true"> </i>
+        Some features are disabled until email is verified.
+        Please verify your email.
```

Comment:
If you change this line, you would need to modify the tests as well (they check for existence/non-existence of `Your email is unverified` (4 instances).

Example: https://github.com/donategifts/donategifts/blob/50356b0a3ab112e4090418e787a8b6cb39171a66/test/wishcards.test.js#L130

Reply:
oh gotcha, thanks for catching this!

Reply:
Np.

There are few instances of `text.should.contain('Your email is unverified');` as well (should have made it clearer earlier, my bad).